EpiAware/src/epiawaremethods/epiawaremethod.jl

```diff
@@ -0,0 +1,11 @@
+"""
```

Comment:
not sure we need this in its own folder? I think everything at the root or of src or in its own submodule makes sense as a pattern? Or do you think there is a good case for more organisation within overarching EpiAware? I agree it makes sense for this functionality to in the sample place as make_epi_aware

Reply:
Yeah ok, happy to move into same place as `make_epi_aware`

Reply:
and your definitely not seeing it as part of `EpiInference`?

Reply:
I think having the overall infrastructure in `EpiAware` but the individual method structs and implementations in EpiInference make sense.

EpiInference is then stand alone, and `EpiAware` is a judgement on a good way to weave the various submodules togethers.

Reply:
> individual method structs and implementations in EpiInference make sense.

I agree this makes sense.

With the current implementation there is nothing `EpiAware` specific about `EpiMethod` is there its just a way of chaining inference approaches together and making sure they will work? 

Given that it seems like you could imagine using them on other inputs that aren't from `EpiAware` and wouldn't it make more sense for it to live in `EpiInference`? i.ee it isn't weaving multiple submodules together all the parts of `EpiMethod` are from `EpiInference`. I totally agree that when it comes time to actually use a model object build out of other parts that is something that lives at the top level.

Reply:
I've moved it into `EpiInference`